cocoapods-plugin/lib/cocoapods-xcremotecache/command/hooks.rb

```diff
@@ -89,61 +89,67 @@ def self.generate_rcinfo()
         @@configuration.select { |key, value| !CUSTOM_CONFIGURATION_KEYS.include?(key) }
       end
 
-      def self.enable_xcremotecache(target, user_proj_directory, xc_location, xc_cc_path, mode, exclude_build_configurations, check_build_configuration, check_platform, final_target)
-          target.build_configurations.each do |config| 
-            # apply only for relevant Configurations
-            next if exclude_build_configurations.include?(config.name)
-            if mode == 'consumer'
-              config.build_settings['CC'] = [xc_cc_path]
-            end
-            config.build_settings['SWIFT_EXEC'] = ["#{xc_location}/xcswiftc"]
-            config.build_settings['LIBTOOL'] = ["#{xc_location}/xclibtool"]
-            config.build_settings['LD'] = ["#{xc_location}/xcld"]
-
-            config.build_settings['XCREMOTE_CACHE_FAKE_SRCROOT'] = FAKE_SRCROOT
-            add_cflags!(config.build_settings, '-fdebug-prefix-map', "#{user_proj_directory}=$(XCREMOTE_CACHE_FAKE_SRCROOT)")
-            add_swiftflags!(config.build_settings, '-debug-prefix-map', "#{user_proj_directory}=$(XCREMOTE_CACHE_FAKE_SRCROOT)")
+      def self.enable_xcremotecache(target, xc_location, xc_cc_path, mode, exclude_build_configurations, final_target)
+        target.build_configurations.each do |config|
+          # apply only for relevant Configurations
+          next if exclude_build_configurations.include?(config.name)
+          if mode == 'consumer'
+            config.build_settings['CC'] = ["$SRCROOT/#{xc_cc_path}"]
           end
-        
+          config.build_settings['SWIFT_EXEC'] = ["$SRCROOT/#{xc_location}/xcswiftc"]
+          config.build_settings['LIBTOOL'] = ["$SRCROOT/#{xc_location}/xclibtool"]
+          config.build_settings['LD'] = ["$SRCROOT/#{xc_location}/xcld"]
```

Comment:
Can you elaborate why `$SRCROOT` are better here? During the build, Xcode will replace `$SRCROOT` to absolute paths anyway and the result will be the same, I think.

Reply:
The issue here is that we are not generating the project and we're commiting the `.pbxproj` to git. Since the plugin is adding full (absolute) paths which work on the machine which integrated the plugin, it will not work on a teammate's machine because the path to the project is different:
<img width="422" alt="Screenshot 2021-11-23 at 00 31 28" src="https://user-images.githubusercontent.com/7620639/142945462-13272ea8-b82e-40a7-9565-aa67ec0b86ee.png">
<img width="768" alt="Screenshot 2021-11-23 at 00 35 11" src="https://user-images.githubusercontent.com/7620639/142945473-367b7711-37a1-46e1-b7e8-08d86ef0401c.png">

Note: My understanding is that the project will be configured once and the changes commited to git.

Reply:
OK, get it.

Maybe committing in the project with engaged XCRemoteCache wasn't how we designed that initially, but I don't see any reason why not to handle it nicely to support your need. Makes sense to me.